tox.ini

```diff
@@ -7,18 +7,16 @@
 
 [tox]
 envlist = py27-swift{1.13.1,2.1.0,2.2.0,2.3.0,2.5.0,2.7.1,2.10.1,2.13.1,2.15.1}
-          py27-swifthead
           pep8
 minversion = 1.8.1
 
 [testenv]
-deps =
+deps = -r{toxinidir}/test-requirements.txt
```

Comment:
Why is this required?

Reply:
No real modification here, it was included line 37 below "swifthead:". I just moved it there for more readability after removing the "swifthead:" line below